.env.example

```diff
@@ -0,0 +1,10 @@
+MASTER_PASSWORD=master_password_changeme
+USER_PASSWORD=admin_password_changeme
+APICAST_REGISTRY_URL=https://apicast-staging.proda.3sca.net/policies
+PROMETHEUS_EXPORTER_PORT=9398
```

Comment:
Sadly you cannot really use that if you launch sidekiq and the webserver at the same time on the same host

Sidekiq will launch its own webserver with this environment variable. And the application webserver will listen to the same port ... so there will be a conflict

Not sure how to solve this with `.env` file without having 2 different env names

Reply:
Do you think this can be made clear?

Reply:
How do you practically use `--index` without introducing another environment variable? This looks more complicated to me to setup and understand than having an alias or letting people manage their own environment variables.

Btw, some of us use aliases:

```shell
alias porta_sidekiq="PROMETHEUS_PORT=9593 bundle exec sidekiq"
alias porta_server="PROMETHEUS_PORT=9394 bundle exec rails s"
```

Also sidekiq 6 removes the `--index` option so I do not really know if we need to commit with that.

I really do not know what dotenv is adding to the project. It was setup at the beginning but I think it is abandoned for reason.

Also it raises the entry barrier by a lot when you need to configure some environments in order to boot the app and make it secure... Especially when this project is already complicated to setup.

Locally, it is fine if the secret is called `secret` as long as bootstrapping the application is not bothersome. I think it is a convention that all applications/softwares have in common. Think about your factory configured router/tv with credentials `admin/admin`. Nobody really expect `some_user/change_me_i_am_not_secure`

Yes for production use you should use something else but not a file either :)